card-gateway/etc/di.xml

```diff
@@ -42,6 +42,7 @@
                 <item name="description" xsi:type="string">Pronko\LiqPayGateway\Gateway\Request\Builder\DescriptionBuilder</item>
                 <item name="order" xsi:type="string">Pronko\LiqPayGateway\Gateway\Request\Builder\OrderBuilder</item>
                 <item name="cardPayment" xsi:type="string">Pronko\LiqPayCardGateway\Gateway\Request\Builder\CardPaymentBuilder</item>
+                <item name="paymentAction" xsi:type="string">Pronko\LiqPayCardGateway\Gateway\Request\Builder\PaymentActionBuilder</item>
```

Comment:
The class namespace is used "LiqPayCardGateway" however the class is located in the "LiqPayGateway" if you can fix this.